### DISCUSSION TITLE
Fix deprecated warnings

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#12119] [Feature Requests]
When you install Directus, you see this deprecations warnings:

```
npm WARN deprecated @types/pino@7.0.5: This is a stub types definition. pino provides its own type definitions, so you do not need this installed.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated fastify-warning@0.2.0: This module renamed to process-warning
npm WARN deprecated @types/cssnano@5.1.0: This is a stub types definition. cssnano provides its own type definitions, so you do not need this installed.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated node-pre-gyp@0.11.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
npm WARN deprecated tar@2.2.2: This version of tar is no longer supported, and will not receive security updates. Please upgrade asap.
```

They are not a bug, but it would be great to remove them.

**Comment by rijkvanzanten** [comment]
A lot of these are dependencies of sub-dependencies of sub-dependencies of sub-dependencies, which makes it quite difficult for us to update them, without opening PRs to all of those dependencies to fix their own dependencies. Even worse; a lot of these are sub-dependencies of _dev_-dependencies, which makes it pointless for us to pursue.. 

I'll move this to a discussion, as it's more an ongoing target than a single fix we can apply

**Reply by rijkvanzanten** [reply]
Oh and while something like dependabot or renovate does help out in keeping dependencies up-to-date, it adds a _tremendous_ amount of maintenance overhead, that doesn't really give us any benefit (if it ain't broke, don't fix it) (with the obvious exception being security issues in dependencies, those should and will always be patched (if they're actually affecting the directus core codebase))

**Comment by rijkvanzanten** [comment]
This has since been addressed
